src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.cpp

```diff
@@ -729,8 +730,12 @@ void SolidMechanicsLagrangianFEM::crsApplyTractionBC( real64 const time,
 
     globalIndex_array nodeDOF;
     real64_array nodeRHS;
-    integer const component = bc.getComponent();
 
+    // The flag component = 0, 1 and 2 are for the traction in the three orthogonal directions
+    // in the global cartesian coordinate system
+    // The flag component = -1 is for the normal traction
+
+    integer const component = bc.getComponent();
```

Comment:
So as I already mentioned, this documentation should be moved to the `doxygen` of `bc`.

Reply:
Please jump in because I don't know how doxygen works yet,

Reply:
The current doc is 
```c++
  /**
   * Accessor
   * @return const m_component
   */
  virtual int getComponent() const
  {
    return m_component;
  }
```
Just enter
```c++
  /**
   * Accessing the considered component.
   * The 0, 1 and 2 values are for the traction in the three orthogonal directions in the global cartesian coordinate system.
   * The -1 value is for the normal traction.
   * @return The component axis or a special value.
   */
  virtual int getComponent() const
  {
    return m_component;
  }
```

Reply:
Tks, done